xml/upgrade-3.10-2.xsl

```diff
@@ -127,6 +127,13 @@
  all resources from the newly transformed nodes.
 -->
 <xsl:template match="node[@type = 'ping']">
+    <xsl:call-template name="info">
+        <xsl:with-param name="msg"
+                        select="concat('Converting ', @uname,
+                                       ' from a ping node to a member node',
+                                       ' with all resources banned')"/>
+    </xsl:call-template>
```

Comment:
I'm leaning against info messages. If behavior is preserved, we don't need any special message, and if it's not, we need a warning.

Reply:
You're probably right. Maybe I was thinking about possible effects on queries/manipulations, but those are supposed to run against the on-disk CIB. If not, we have a bug.

Reply:
Ah, I remember now. I was thinking about the effects on ACLs (and maybe other things but I think that's all). ACLs configured to apply to ping nodes or master resources would change their behavior after two of these transformations, for example.

Granted, that would be a bizarre ACL match...

Reply:
Let's address that in the ACL PR and leave this one for warnings